common/src/main/java/com/wynntils/overlays/PartyMembersOverlay.java

```diff
@@ -238,6 +238,22 @@ public void render(
                     healthTexture.getTextureY2(),
                     (float) healthProgress);
 
+            if(healthProgress > 1.01) {
```

Comment:
```suggestion
            if (Math.abs(healthProgress) > 1) {
```

Just to keep it using the same logic as the normal overflow bars